tailcfg/tailcfg.go

```diff
@@ -259,7 +259,7 @@ type Service struct {
 	_           structs.Incomparable
 	Proto       ServiceProto // TCP or UDP
 	Port        uint16       // port number service is listening on
-	Description string       // text description of service
+	Description string       `json:",omitempty"` // text description of service
```

Comment:
Based on my reading of `xcode/Shared/IPN.swift`, these changes should be safe, but please do give it a try.